plugin/storage/integration/integration.go

```diff
@@ -40,27 +40,34 @@ import (
 )
 
 const (
-	iterations = 100
+	iterations  = 100
+	maxSpanTime = 72 * time.Hour
```

Comment:
```suggestion
	maxSpanAge = 72 * time.Hour
```

Reply:
Done

Reply:
Initially it was directly defined itself, but in es_test it was defined as constant, that's why changed it to constant here as well